samples/client/petstore/lua/spec/api_response_spec.lua

```diff
@@ -1,23 +1,21 @@
 --[[
-Swagger Petstore
```

Comment:
Was indentation change on purpose?

Reply:
Yup, I've not manually deleted the spec files as spec files won't be overwritten during code generation.